src/index.js

```diff
@@ -8,7 +8,7 @@ export default class extends XHRBackend {
       Object.assign(
         {
           ajax: (url, { init } = {}, cb) => axios(url, Object.assign({
-            transformResponse: undefined, // i18next expects JSON, not an object
+            transformResponse: [(data) => data], // i18next expects JSON, not an object
```

Comment:
Passing an empty array may also work, but it seems better to explicitly return untransformed data and this is a tested fix.

Reply:
should also be `data => data`, just fixed that up